### DISCUSSION TITLE
Using Redis with WatsonORM

### DISCUSSION THREAD

**Discussion by ghost** [jchristn/WatsonORM#8] [General]
Are there any future plans to include redis into watson orm to boost performace? Or are there any caching methods already included?

**Comment by jchristn** [comment]
Hi @flo2000ace I didn't want to build or integrate anything directly into the library that could potentially create data coherency issues.  As a result there are no plans to integrate Redis or add caching capabilities of any kind.  That said the DBMS supported by WatsonORM all have their own native caching capabilities which do help improve performance, though it doesn't mitigate the roundtrip of a query/response.  Cheers
